src/js/components/footer.js

```diff
@@ -0,0 +1,41 @@
+import { newText, newMultimedia, newLink, newContainer } from './helpers';
+
+const createlinks = (datalinks) =>{
+  const arraylinks = datalinks.map((link)=>{
```

Comment:
@valuarte What your team meant with lowercase was just the `LI` in the function `newContainer` not the variable names. The variables should be camel case, like it was before.

Reply:
I already organized style attributes alphabetically